analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AppSettings/DoNotHardcodeCredentials/Valid/AppSettings.json

```diff
@@ -49,7 +49,13 @@
     {
       "Password": true
     }
-  ]
+  ],
+  "NumberFields": {
+    "Field1": 42,
+    "Field2": -42,
+    "Field3": 2147483648,
+    "Password": 2147483648 /* FN */
```

Comment:
This is something to be detected right?
In case I"ll open an FN.

Reply:
Yes, I think so.
Maybe the suggested changes around `object BuildResult()` may improve things.

I would also add `int.MinValue-1`, `long.MaxValue + 1` and `long.MinValue - 1` to the list of cases.